src/util/tools.h

```diff
@@ -34,11 +34,7 @@
 #include <chrono>
 #include <filesystem>
 #include <map>
-#include <memory>
-#include <optional>
```

Comment:
This is still needed.

```
src/util/tools.h:162:6: error: no template named 'optional' in namespace 'std'
std::optional<std::vector<std::byte>> readBinaryFile(const fs::path& path);
```

Reply:
Seems to heavily depend on the compiler. Readded